styles/tokens/input.json

```diff
@@ -132,29 +123,104 @@
       "file": "settings",
       "themable": true
     },
-    "focus-border-color": {
-      "comment": "The border color on Inputs when focused.",
-      "value": "{purple.value}",
-      "file": "settings",
-      "themable": true
-    },
-    "error-border-color": {
-      "comment": "The border color on Inputs when there is an error.",
-      "value": "{orange.value}",
-      "file": "settings",
-      "themable": true
+    "focus": {
+      "border-color": {
+        "comment": "The border color on Inputs when focused.",
+        "value": "{purple.value}",
+        "file": "settings",
+        "themable": true
+      },
+      "border-width": {
+        "comment": "The border width on Inputs when focused.",
+        "value": "2px",
+        "attributes": {
+          "category": "size"
+        },
+        "file": "settings",
+        "themable": true
+      },
+      "box-shadow": {
+        "comment": "The box shadow on Inputs when focused.",
+        "value": "0 0 0 2px {purple.value}, 0 2px 8px {black-tint.30.value}",
+        "attributes": {
+          "category": "content"
+        },
+        "file": "settings",
+        "themable": true
+      }
     },
-    "disabled-border-color": {
-      "comment": "The border color on disabled Inputs.",
-      "value": "{black-tint.50.value}",
-      "file": "settings",
-      "themable": true
+    "error": {
+      "box-shadow": {
+        "comment": "The box shadow on Inputs there is an error.",
+        "value": "0 0 0 1px {orange.value}",
+        "attributes": {
+          "category": "content"
+        },
+        "file": "settings",
+        "themable": true
+      },
+      "focus-box-shadow": {
+        "comment": "The box shadow on Inputs there is an error and it's focused.",
+        "value": "0 0 0 1px {orange.value}, 0 2px 8px {black-tint.30.value}",
+        "attributes": {
+          "category": "content"
+        },
+        "file": "settings",
+        "themable": true
+      },
+      "border-width": {
```

Comment:
Let's take a look at this -- we got rid of all borders on inputs so this might not be necessary